### DISCUSSION TITLE
V5我想把interval柱状图设置固定宽度，但是都没找到类似配置项

### DISCUSSION THREAD

**Discussion by fjloser** [antvis/G2#6000] [Q&A]
### 问题描述

V5我想把interval柱状图设置固定宽度，但是我都没找到类似配置项，请问可以在那个地方可以配置呢？

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by hustcc** [comment]
设置 style maxWidth minWidth

**Comment by pearmini** [comment]
如果是固定宽度，可以设置为 `chart.interval().encode("size", 50)`
